VideoWeb/VideoWeb.EventHub/Handlers/Core/EventHandlerBase.cs

```diff
@@ -48,15 +48,15 @@ public virtual async Task HandleAsync(CallbackEvent callbackEvent)
         /// <param name="newStatus"></param>
         /// <returns></returns>
```

Comment:
just passing through; could update summary to include `reason` param

Reply:
done